.vscode/launch.json

```diff
@@ -0,0 +1,101 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "lldb",
+            "request": "launch",
+            "name": "Debug unit tests in library 'chombo-gen-common'",
+            "cargo": {
+                "args": [
+                    "test",
+                    "--no-run",
+                    "--lib",
+                    "--package=chombo-gen-common"
+                ],
+                "filter": {
+                    "name": "chombo-gen-common",
+                    "kind": "lib"
+                }
+            },
+            "args": [],
+            "cwd": "${workspaceFolder}"
+        },
+        {
+            "type": "lldb",
+            "request": "launch",
+            "name": "Debug executable 'chombo-gen-backend'",
+            "cargo": {
+                "args": [
+                    "build",
+                    "--bin=chombo-gen-backend",
+                    "--package=chombo-gen-backend"
+                ],
+                "filter": {
+                    "name": "chombo-gen-backend",
+                    "kind": "bin"
+                }
+            },
+            "args": [],
+            "cwd": "${workspaceFolder}"
+        },
+        {
+            "type": "lldb",
+            "request": "launch",
+            "name": "Debug unit tests in executable 'chombo-gen-backend'",
+            "cargo": {
+                "args": [
+                    "test",
+                    "--no-run",
+                    "--bin=chombo-gen-backend",
+                    "--package=chombo-gen-backend"
+                ],
+                "filter": {
+                    "name": "chombo-gen-backend",
+                    "kind": "bin"
+                }
+            },
+            "args": [],
+            "cwd": "${workspaceFolder}"
+        },
+        {
+            "type": "lldb",
+            "request": "launch",
+            "name": "Debug executable 'chombo-gen-frontend'",
```

Comment:
Does this even work? I thought it just generates wasm binary, which is not usable outside of web browser's context.